src/App/AppRoute.tsx

```diff
@@ -36,36 +36,34 @@ const AppRoute: React.FC = () => {
 
   return (
     <Router history={history}>
-      <Suspense fallback={<Loading />}>
```

Comment:
I need you to explain why you are moving `<Suspense/>`  from top-level to two different parts of the app, Header and App ?

Reply:
Ok, I think you forgot to remove the `<Suspense/>` from `Header`.